tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowList.java

```diff
@@ -28,19 +29,29 @@ public class WorkflowList {
         this.ga4Ghv20Api = ga4Ghv20Api;
         this.extendedGa4GhApi = extendedGa4GhApi;
+        this.workflowsApi = workflowsApi;
+        this.configFilePathWDL = configFilePathWDL;
+        this.configFilePathCWL = configFilePathCWL;
+
         this.workflowsToTest = getDefaultTests();
     }
     private final List<WorkflowRunner> workflowsToTest;
 
     private List<WorkflowRunner> getDefaultTests() throws InterruptedException {
-        return List.of(new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "test-parameter-files/agc-fastq-read-counts-test-parameter-file.json", extendedGa4GhApi),
-                        new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "agc-examples/fastq/input.json", ga4Ghv20Api, extendedGa4GhApi),
-                        new WorkflowRunner("github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM", "3.0.0", "test-parameter-files/BAM-to-Unmapped-BAM-test-parameter-file.json", extendedGa4GhApi),
-                        new WorkflowRunner("github.com/manning-lab/vcfToGds", "main", "test-parameter-files/vcfToGds-test-parameter-file.json", extendedGa4GhApi),
-                        new WorkflowRunner("github.com/DataBiosphere/analysis_pipeline_WDL/assocation-aggregate-wdl", "v7.1.1", "test-parameter-files/assocation-aggregate-wdl-test-parameter-file.json", extendedGa4GhApi)
+        return List.of(new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "test-parameter-files/agc-fastq-read-counts-test-parameter-file.json", extendedGa4GhApi, workflowsApi, configFilePathWDL, configFilePathCWL),
+                        new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "agc-examples/fastq/input.json", ga4Ghv20Api, extendedGa4GhApi, workflowsApi, configFilePathWDL, configFilePathCWL),
+                        new WorkflowRunner("github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM", "3.0.0", "test-parameter-files/BAM-to-Unmapped-BAM-test-parameter-file.json", extendedGa4GhApi, workflowsApi, configFilePathWDL, configFilePathCWL),
+                        new WorkflowRunner("github.com/manning-lab/vcfToGds", "main", "test-parameter-files/vcfToGds-test-parameter-file.json", extendedGa4GhApi, workflowsApi, configFilePathWDL, configFilePathCWL),
+                        new WorkflowRunner("github.com/DataBiosphere/analysis_pipeline_WDL/assocation-aggregate-wdl", "v7.1.1", "test-parameter-files/assocation-aggregate-wdl-test-parameter-file.json", extendedGa4GhApi, workflowsApi, configFilePathWDL, configFilePathCWL),
+                        new WorkflowRunner("github.com/fhembroff/wes-testing/nontrivial", "main", "test-parameter-files/nontrivial-test-parameter-file.json", extendedGa4GhApi, workflowsApi, configFilePathWDL, configFilePathCWL),
```

Comment:
Can also be done in that follow-up ticket, these should probably be tagged to be more resilient against future accidental changes

Reply:
I created this tag for my workflows `stable-version-for-testing-v1`.